### PR TITLE
Delete tmp dir on exit

### PR DESCRIPTION
**Pull request by uriya222**

### CODE REVIEW

File: AstroPhotometry/View/MainWindow.xaml.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 using System.Windows.Input;
```
Comment (ishay320) [inline AstroPhotometry/View/MainWindow.xaml.cs:8]: remove line

File: AstroPhotometry/View/MainWindow.xaml.cs [hunk 2]
```diff
@@ -49,7 +50,27 @@ private void Button_Click(object sender, RoutedEventArgs e)
                 photo.updateUri(openFileDialog.FileName);
             }
             //txtEditor.Text = File.ReadAllText(openFileDialog.FileName);
-        } 
+        }
+        private void Window_Closing(object sender, CancelEventArgs e)
```
Comment (ishay320) [inline AstroPhotometry/View/MainWindow.xaml.cs:54]: add line between functions

File: AstroPhotometry/View/MainWindow.xaml.cs (hunk 1 above)
Comment (ishay320) [inline AstroPhotometry/View/MainWindow.xaml.cs:8]: remove line

**Review by ishay320 — APPROVED** [review]

File: AstroPhotometry/View/MainWindow.xaml.cs (hunk 2 above)
Comment (ishay320) [reply AstroPhotometry/View/MainWindow.xaml.cs:54]: and remove  the _

**Review by ishay320 — APPROVED** [review]

File: AstroPhotometry/View/MainWindow.xaml.cs (hunk 2 above)
Comment (uriya222) [reply AstroPhotometry/View/MainWindow.xaml.cs:53]: See one function above - you are using it as well

**Pull request merged**